index.js

```diff
@@ -189,8 +223,14 @@ async function getSize({baseRef, exec, excludes}) {
 			}
 		})
 
+	let additionalIgnores = []
+	if (ignoreDeletedLines) {
+		additionalIgnores = data.filter(d => d.added === 0 && d.removed > 0).map(d => d.name)
+	}
+
 	return {
-		size: data.reduce((t, d) => t + d.added + d.removed, 0),
-		includes: data.map(d => d.name)
+		size: data.reduce((t, d) => t + (ignoreDeletedLines ? d.added : d.added + d.removed), 0),
+		includes: data.map(d => d.name),
+		ignores: additionalIgnores,
```

Comment:
@matdurand I'm just going through some pre-release testing and it looks great except for one scenario. If ignore-deleted-lines=true and ignore-deleted-files is false, a deleted file appears in both includes and ignores. To fix, I think we can do something like:

```js
return {
	size: data.reduce((t, d) => t + (ignoreDeletedLines ? d.added : d.added + d.removed), 0),
	includes: data.map(d => d.name).filter(f => !additionalIgnores.contains(f)),
	ignores: additionalIgnores,
}
```

Would that change be a problem for you?

Reply:
Yeah, all good!